src/main/java/net/dv8tion/jda/core/utils/Helpers.java

```diff
@@ -0,0 +1,123 @@
+/*
+ *     Copyright 2015-2017 Austin Keener & Michael Ritter & Florian Spieß
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.core.utils;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Objects;
+
+/**
+ * This class has major inspiration from <a href="https://commons.apache.org/proper/commons-lang/" target="_blank">Lang 3</a>
+ *
+ * <p>Specifically StringUtils.java and ExceptionUtils.java
+ */
+public final class Helpers
+{
+
+    // ## StringUtils ##
+
+    public static boolean isEmpty(final CharSequence seq)
+    {
+        return seq == null || seq.length() == 0;
+    }
+
+    public static boolean containsWhitespace(final CharSequence seq)
+    {
+        if (isEmpty(seq))
+            return false;
+        for (int i = 0; i < seq.length(); i++)
+        {
+            if (Character.isWhitespace(seq.charAt(i)))
+                return true;
+        }
+        return false;
+    }
+
+    public static boolean isBlank(final CharSequence seq)
+    {
+        if (isEmpty(seq))
+            return true;
+        for (int i = 0; i < seq.length(); i++)
+        {
+            if (!Character.isWhitespace(seq.charAt(i)))
+                return false;
+        }
+        return true;
+    }
+
+    public static boolean equalsIgnoreCase(final String seq0, final String seq1)
+    {
+        return Objects.equals(seq0, seq1) || (seq0 != null && seq0.equalsIgnoreCase(seq1));
+    }
+
+    public static int countMatches(final CharSequence seq, final char c)
+    {
+        if (isEmpty(seq))
+            return 0;
+        int count = 0;
+        for (int i = 0; i < seq.length(); i++)
+        {
+            if (seq.charAt(i) == c)
+                count++;
+        }
+        return count;
+    }
+
+    public static String truncate(final String input, final int maxWidth)
+    {
+        if (input == null)
+            return null;
+        Checks.notNegative(maxWidth, "maxWidth");
+        if (input.length() == maxWidth)
```

Comment:
Thank you for pointing that out, apparently I haven't tested this with too long width.